llvm/lib/Target/AMDGPU/VOPCInstructions.td

```diff
@@ -454,6 +454,11 @@ multiclass VOPCX_Pseudos <string opName,
     let isCommutable = 1;
     let SubtargetPredicate = HasNoSdstCMPX;
     let IsVCMPX = 1;
+    let AsmMatchConverter =
+        !if (P_NoSDst.HasOpSel, "cvtVOP3OpSel",
+            !if (!or(P_NoSDst.HasModifiers, P_NoSDst.HasOMod,
+                     P_NoSDst.HasIntClamp),
+                 "cvtVOP3", ""));
```

Comment:
Nit: could a `!cond()` be more readable here?

```suggestion
    let AsmMatchConverter = !cond(
        P_NoSDst.HasOpSel : "cvtVOP3OpSel",
        !or(P_NoSDst.HasModifiers, P_NoSDst.HasOMod, P_NoSDst.HasIntClamp) : "cvtVOP3",
        1 : "");
```

Reply:
agree

Reply:
replaced all with !cond